ui/pages/sample/components/sample-counter-pane.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Box, Button, Text } from '../../../components/component-library';
+import Card from '../../../components/ui/card';
+import {
+  Display,
+  FlexDirection,
+  AlignItems,
+  TextAlign,
+  TextVariant,
+  BlockSize,
+} from '../../../helpers/constants/design-system';
+import { useSample } from '../../../ducks/sample';
+
+export function SampleCounterPane() {
+  const counter = useSample();
+
+  return (
+    <Card
+      data-testid="sample-counter-pane-card"
+      width={BlockSize.Full}
+      style={{ maxWidth: '500px' }}
```

Comment:
Nit: We should not use inline styles

Reply:
Agree! If there isn’t a style utility prop that can be used, then we should fall back to attaching a class name and adding an SCSS stylesheet. Inline styles should be avoided where possible or used only when we need dynamic values. This helps with CSS specificity and, I believe, also provides some performance benefits.